README.rst

```diff
@@ -32,6 +32,17 @@ Key Features
 - Proper rate limit handling
 - Optimised in both speed and memory
 
+Reasons to use Nextcord over alternatives
+-------------
+
+* No breaking changes unless necessery
+* A lot of bug fixes
+* Interactions (which are currently under development)
+* Compatibility with libraries such as:
+
+  * `jishaku <https://github.com/Gorialis/jishaku>`_
+  * `dislash <https://github.com/EQUENOS/dislash.py>`_
```

Comment:
:+1: Looks good

A few suggestions:

- maybe make heading shorter
- remove extra words
- keep bullet-style consistent with other lists (`-` instead of `*`)
- Possibly "compatible libraries" could be a section on its own?

```suggestion
Why choose Nextcord?
--------------------

- No breaking changes unless necessary
- Lots of bug fixes
- Interactions (currently under development)
- Compatibility with libraries such as:

  - `jishaku <https://github.com/Gorialis/jishaku>`_
  - `dislash <https://github.com/EQUENOS/dislash.py>`_
```